sktime/annotation/tests/test_stray.py

```diff
@@ -0,0 +1,274 @@
+# -*- coding: utf-8 -*-
+"""Tests for STRAY outlier estimator."""
+
+__author__ = ["KatieBuc"]
+
+import numpy as np
+
+from sktime.annotation.stray import STRAY, standardize
+
+
+def test_default_1D():
+    """Test with default parameters and 1D input array."""
+    X = np.array(
```

Comment:
can you use `make_forecasting_problem` to create test data?

Reply:
ahhh I created these arrays in R, and checked directly against the R output of the original implementation - I could include (as a comment) the R code so that developers know the context of this input data? Or you still think it's best to sever the connection to the R implementation and use an sktime data generator?